doc/man/cockpit.conf.xml

```diff
@@ -196,6 +196,20 @@ ProtocolHeader = X-Forwarded-Proto
     </variablelist>
   </refsect1>
 
+  <refsect1 id="cockpit-conf-session">
+    <title>Session</title>
+    <variablelist>
+      <varlistentry>
+        <term><option>IdleTimeout</option></term>
+        <listitem>
+          <para>Time in minutes after which session expires and user is logged out if no user action
+          has been performed in the given time. Set to <literal>0</literal> to disable session timeout.
+          When not specified, the default is <literal>15</literal> minutes.</para>
```

Comment:
Shouldn't the default be `0`?

Reply:
No, we think that it should be ON by default. Do you think otherwise?

Reply:
Yes, the CC requirement sounds like it needs to be on by default.